dart/lib/src/sentry.dart

```diff
@@ -155,6 +162,8 @@ class Sentry {
     final hub = currentHub;
     _hub = NoOpHub();
     hub.close();
+
+    _disposeIsolate?.call();
```

Comment:
this should be in the `hub.close()` method, see:

https://github.com/getsentry/sentry-java/blob/2e3679d18ec8be209fb5ad1432fc26c4adf2a268/sentry/src/main/java/io/sentry/Hub.java#L278-L282

ideally, as List, other integrations might have the same problem

Reply:
✅